src/main/java/org/radarcns/service/SourceService.java

```diff
@@ -24,6 +29,8 @@
 
     private final ManagementPortalClient managementPortalClient;
 
+    private static final Duration DEFAULT_SOURCE_CONNECTION_TIMEOUT = Duration.ofHours(6);
```

Comment:
How was this decided? Was it chosen at random?

Reply:
The default value is set to 6 HOURS and yes it is random. It is only effective if a timeout is not configured for a source-type